build.rs

```diff
@@ -16,6 +16,30 @@ pub fn gdal_version_info(key: &str) -> String {
     }
 }
 
+#[cfg(not(docsrs))]
+pub fn _gdal_create_geometry(wkt: &str) -> gdal_sys::OGRGeometryH {
+    let c_wkt = std::ffi::CString::new(wkt).expect("Ooops, cannot create CString !");
+    let mut c_wkt_ptr = c_wkt.into_raw();
```

Comment:
I think this leaks memory, see #172

Reply:
Pushed fix.

Reply:
Are you sure? I still see an `into_raw()` call instead of `.as_ptr()`.

Reply:
fixed `into_raw` -> `as_ptr`